cogs/auctions.py

```diff
@@ -457,7 +457,7 @@ def check(m):
                 self.bot.loop.create_task(
                     self.bot.send_dm(
                         auction.bidder_id,
-                        f"You have been outbid on the **{auction.pokemon.iv_percentage:.2%} {auction.pokemon.species}** with a Current bid of **{auction.current_bid:,}** Pokécoins (Auction #{auction.id}).",
+                        f"You have been outbid on the **{auction.pokemon.iv_percentage:.2%} {auction.pokemon.species}** with a Current Bid of **{auction.current_bid:,}** Pokécoins (Auction #{auction.id}).",
```

Comment:
```suggestion
f"You have been outbid on the **{auction.pokemon.iv_percentage:.2%} {auction.pokemon.species}** with a current bid of **{auction.current_bid:,}** Pokécoins (Auction #{auction.id}).",
```
this isn't meant to be capitalized, its in the middle of a sentence.

Reply:
ah, i capitalized that for emphasis since "Current Bid:" is how it's shown on the auction info, but i understand